.github/workflows/integration_test.yml

```diff
@@ -113,7 +113,11 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.ruby-version }}
+          rubygems: latest
           bundler-cache: true
+      - name: Update RubyGems
```

Comment:
You can remove that section now since it's taken care of with `rubygems: latest`